Makefile

```diff
@@ -104,6 +104,8 @@ build-portal:
 	cd portal && npm install && npm run build
 
 pyenv:
+	sudo dnf install -y python3-devel && \
```

Comment:
We'll want to move this to the CI VM provisioning script rather than assuming the package manager on the user's laptop as well as a `sudo` command.